src/Microsoft.Health.SqlServer/SqlConnectionBuilder.cs

```diff
@@ -0,0 +1,55 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Data.SqlClient;
+using Microsoft.Health.SqlServer.Configs;
+
+namespace Microsoft.Health.SqlServer
+{
+    /// <summary>
+    /// Helper class to build the base SqlConnection object
```

Comment:
>SqlConnection [](http://example.com/codeflow?start=39&length=13)

supernit:
you can use <see cref="SqlConnection"/> and VS and any other editor would pick it up in case of renaming the class.

Otherwise it's just a string, and you need to update string manually in case of refactoring.